packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2862,6 +2864,17 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     markNeedsSemanticsUpdate(onlyChanges: (value != null) == hadValue);
   }
 
+  /// If non-null, sets the "isSelected" semantic to the given value.
```

Comment:
hyperlink to the actual isSelected member instead of just putting it in quotes. (i.e. put the fully qualified name in square brackets)